### DISCUSSION TITLE
Release V1.0.12

### DISCUSSION THREAD

**Discussion by moabu** [JanssenProject/jans#4499] [Polls]
https://github.com/JanssenProject/jans/milestone/15

**Comment by ossdhaval** [comment]
@moabu What date are we targeting for 1.0.12? Let's start preparing for it a few days in advance. We need to 
- identify fixes that are **must have** for this release
- give a couple of days of heads-up to the test team to certify

**Reply by moabu** [reply]
Yep that's the idea. The date is mentioned in the milestone attached in the poll.

**Comment by moabu** [comment]
- [ ] https://artifacthub.io/packages/helm/janssen-auth-server/janssen?modal=security-report all reported high and critical issues needs to be addressed before releasing.

**Reply by yuriyz** [reply]
jettison is upgraded on AS side https://github.com/JanssenProject/jans/commit/e90269fb58f021377d098a45b38a6ba0fc9220d1

**Comment by ossdhaval** [comment]
Looking at the current security report, I am voting `No`. Excluding OpenDJ, the report shows 1 Critical and 8 High. We need to cover this ground by Thursday (Release being on Friday).

@manojs1978 Can you please update the status of testing here?

**Reply by manojs1978** [reply]
sanity test is PASSED,

------------------------------------------------------------------------------------------------------------------------
| \#  | Item to Test                                                     |  status |  comment                          |
| --- | -----------------------------------------------------------------|---------|-----------------------------------|
| 1   |  RHEL8 +(opendj, mysql + pgsql)installtion                       |  PASS   |                                   |
| 2   |  SUSE 15 +(mysql + pgsql)installtion                             |  PASS   |  open issue--openDJ installation        |
| 3   |  UBUNTU 20 + (opendj, mysql + pgsql)installtion                  |  PASS   |                                   |
| 4   | UBUNTU 22 + (opendj, mysql + pgsql)installtion                   |  PASS   |                                   |
| 5   |  Review .well-known config files: openid, fido, uma, scim        |  PASS   |                                   |
| 6   |Start the TUI and use device-flow for login                       |  PASS   |                                   |
| 7   | Jans Tent: password authentication, acr=simple_password_auth     |  PASS   |       manually added scope and completed  flow  |
| 8   |Jans Tent: test Agama Project deployment,acr=agama&agama_flow=qa1 |  PASS   |   manually added scope and completed  flow      |
------------------------------------------------------------------------------------------------------------------------

**Comment by moabu** [comment]
security vulns has dropped and critical vulns have been resolved in the auth server. This currently does pass the auth release gate. Due to the urgency of this release we will release it and follow it with a security patch in 1-2 weeks.

**Comment by moabu** [comment]
https://github.com/JanssenProject/jans/releases/tag/v1.0.12
